src/stories/info/info.stories.tsx

```diff
@@ -1,6 +1,9 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
```

Comment:
Same question : Why not using the newer syntax ? https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-require-imports.md